### DISCUSSION TITLE
Mention what is covered by semantic versioning (GUI vs. CLI vs. RPCs vs. file formats, etc)

### DISCUSSION THREAD

**Discussion by altendky** [olivierlacan/keep-a-changelog#536] [Ideas]
https://github.com/olivierlacan/keep-a-changelog/blob/ee8cc0c6256716bb57af41f7e58001fb58ae193d/source/en/1.0.0/index.html.haml#L81

I think the intent here is valuable but without defining what is considered part of the "API" it leaves the reader to make assumptions that may well differ from the developer's plan for backwards compatibility.

https://github.com/semver/semver/blame/d5d9e738e3e04df4979dd0e95e8f860003f2f46e/semver.md#L9
```
1. MAJOR version when you make incompatible API changes
```

I would suggest at least adding something along the lines of "and what interfaces are covered".  If there's interest in this, I can think the wording over and provide a PR.  Or you can, no matter to me.

**Comment by olivierlacan** [comment]
That seems like a good idea. Do you mind suggesting this change as a PR targeting version `en/1.2.0/` (you'll likely need a new Haml file if one hasn't been created yet by the time you fetch) since any significant phrase change will need to go through the translation tracking process?

**Comment by altendky** [comment]
Hmm, no autolink, so... https://github.com/olivierlacan/keep-a-changelog/pull/540.
